src/pages/tezos/academy/index.js

```diff
@@ -95,6 +104,10 @@ const CurriculumOverview = ({
     return p;
   }, []);
 
+  useEffect(() => {
+    trackEvent('Academy-View');
```

Comment:
You don't have to specifically track a particular view. Amplitude does that by default.

Reply:
Other than this. Everything looks great.

Reply:
Okay. But afaik, specific views were being tracked manually in the v1 code. 
Any reason behind that?